_articles/finding-users.md

```diff
@@ -19,15 +19,15 @@ related:
 
 ## Spreading the word
 
-There's no rule that says you have to promote an open source project when you launch. There are many fulfilling reasons to work in open source that have nothing to do with popularity. If you are hoping others will find and use your open source project, however, it's time to tell everybody about your hard work!
+There's no rule that says you have to promote an open source project when you launch. There are many fulfilling reasons to work in open source that have nothing to do with popularity. Instead of hoping others will find and use your open source project, you have to _spread the word_ about your hard work!
```

Comment:
This would be better without the emphasis. The exclamation point does the job.

Reply:
Will remove the emphasis